velox/functions/prestosql/registration/JsonFunctionsRegistration.cpp

```diff
@@ -15,10 +15,11 @@
  */
 
 #include "velox/functions/Registerer.h"
-#include "velox/functions/prestosql/JsonExtractScalar.h"
+#include "velox/functions/prestosql/JsonFunctions.h"
 
 namespace facebook::velox::functions {
 void registerJsonFunctions() {
+  registerFunction<isJsonScalarFunction, bool, Varchar>({"is_json_scalar"});
```

Comment:
Uppercase 'I' here. That is the convention for function structures.

Reply:
Thanks for sharing, modified accordingly